### ISSUE TITLE
Brand new v5 install, error on first startup - Error: Cannot find module '@strapi\plugin-content-manager\package.json' 

### ISSUE THREAD

**Issue by jbeckton** [strapi/strapi#20164]
### Required System information

- Node.js version: v20.12.2
- NPM version: 10.5.0
- Strapi version: 5.0.0-beta.4
- Database: Postgres
- Operating system: Windows 11
- Is your project Javascript or Typescript: Typescript

When I run strapi report I get the following error.

![image](https://github.com/strapi/strapi/assets/2669585/ddc6af39-4e2d-4c65-8a96-f5dd464cdf08)

### Describe the bug

After generating my first strapi v5 app locally using the cli command `npx create-strapi-app@beta strapi` choosing the custom option as well as typescript and then providing my postgres db info. When I try and run the new strapi instance for the first time using `npm run develop` I get missing module errors. The modules that it is complaining about are plugins such as `@strapi\plugin-content-manager`.

Over the course of the last hour, I have tried to re-install and run strapi v5 a few times and have seen random missing module errors. These missing modules have all been from `@strapi/plugin-*` 

### Steps to reproduce the behavior

1. `npx create-strapi-app@beta strapi`
2. `cd strapi`
3. `npm run develop`

### Expected behavior

I would expect strapi to start up and allow me to finish setting up the local instance.

### Screenshots

![image](https://github.com/strapi/strapi/assets/2669585/a4d4b5d5-a8f3-44de-95a2-08c03fde73de)

Here is the package.json file that the cli created.

![image](https://github.com/strapi/strapi/assets/2669585/ae8b8f0c-3038-44bb-9350-8b128a6033d0)

In previous versions of strapi I did not have to add these core plugins via npm install.

**Comment by jbeckton** [comment]
TO add some context to this issue. I created two instances of strapi locally using the steps described in the initial post. Niether would start up due to missing modules. I added the missing modules to the package.json file for both instances.

````
    "@strapi/plugin-content-manager": "5.0.0-beta.4",
    "@strapi/plugin-email": "5.0.0-beta.4",
````

The first instance will not start up and allow me to complete the installation. The second instance will not. The difference in the two instances is that the one that will not run is located inside a mono-repo manage by Turborepo.

There are possibly two issues here. One is that the cli did not set up the package.json correctly for both instances, hence the missing packages.

The second issue is that even after adding the missing packages for the second instance (mono-repo) the loaders do not have the correct path when search to load these modules.

I can run strapi v4 inside a mono-repo as many times as I want but with v5 it does not seem to work. I am unable to load the strapi instance for the first time to complete the set up in the second instance (mono-repo).

**Comment by derrickmehaffy** [comment]
Can't reproduce in beta.5

**Comment by konstantinkreft** [comment]
I guess I'm having the same issue with a fresh install of strapi@beta. But for me it is the email plugin which is missing:

```
Error: Cannot find module '@strapi/plugin-email/package.json'      
```

All I did before was `npx create-strapi-app@beta strapi-5-beta --ts`

Required System information:
- Node.js version: v20.12.2
- NPM version: 10.5.0
- Strapi version: 5.0.0-beta.5
- Database: SQlite
- Operating system: macOS 14.4.1
- Is your project Javascript or Typescript: Typescript

**Comment by jbeckton** [comment]
@konstantinkreft  Right, and once you add the email plugin you will then see the next missing plugin. It seems to be the core plugin modules that are the issue. I have a monorepo set up. I installed strapi 5 outside of the monorepo and then copied over the missing modules from that into the monorepo from the node_modules folder.

**Comment by jbeckton** [comment]
> Can't reproduce in beta.5

Are you using monorepo set up for your test?

**Comment by derrickmehaffy** [comment]
We dont test using monorepos that have some kind of centralized package management as we don't support them for projects. Eg we do have internal monorepos for projects but we set them up as split package management: 

https://GitHub.com/Strapi/internal-lms
https://GitHub.com/Strapi/nextjs-corporate-starter

**Comment by konstantinkreft** [comment]
It is not only a problem with monorepo setups. I have no monorepo involved and still get these errors on a fresh install. 

I don't have `yarn` installed nor do I plan to. Can this problem be related to the fact that I am using plain `npm`?

@jbeckton Are you also using plain `npm`?

**Comment by jbeckton** [comment]
@konstantinkreft  Yes I use NPM only.

**Comment by jbeckton** [comment]
Strapi v4 worked fine within monorepo. v5 does not.

**Comment by gausoft** [comment]
> I guess I'm having the same issue with a fresh install of strapi@beta. But for me it is the email plugin which is missing:
> 
> ```
> Error: Cannot find module '@strapi/plugin-email/package.json'      
> ```
> 
> All I did before was `npx create-strapi-app@beta strapi-5-beta --ts`
> 
> Required System information:
> 
> * Node.js version: v20.12.2
> * NPM version: 10.5.0
> * Strapi version: 5.0.0-beta.5
> * Database: SQlite
> * Operating system: macOS 14.4.1
> * Is your project Javascript or Typescript: Typescript

I have the same issue

**Comment by gausoft** [comment]
After fresh install it's crashes on rollup.
![cannot-find-module-rollup](https://github.com/strapi/strapi/assets/26385667/d369799b-d5cb-41a5-a8dc-d0b1fe301ee3)
When I enter `npm install`, I get "email plugin missing" now
![cannot-find-module-strapi-plugin-email](https://github.com/strapi/strapi/assets/26385667/cc916506-dc71-4b0f-81cc-66e368fa7936)

**Comment by Sam-Flexiti** [comment]
> I guess I'm having the same issue with a fresh install of strapi@beta. But for me it is the email plugin which is missing:
> 
> ```
> Error: Cannot find module '@strapi/plugin-email/package.json'      
> ```
> 
> All I did before was `npx create-strapi-app@beta strapi-5-beta --ts`
> 
> Required System information:
> 
> * Node.js version: v20.12.2
> * NPM version: 10.5.0
> * Strapi version: 5.0.0-beta.5
> * Database: SQlite
> * Operating system: macOS 14.4.1
> * Is your project Javascript or Typescript: Typescript

This issue still happens in beta6, it is missing this email plugin. When I tried earlier today, there is not beta6 version of the email plugin. I'm staying with beta5 for now. 

npm i —save @strapi/plugin-email@5.0.0-beta.5

**Comment by derrickmehaffy** [comment]
We had a small issue this morning where some packages didn't get published but that should have been resolved.

**Comment by gausoft** [comment]
> > I guess I'm having the same issue with a fresh install of strapi@beta. But for me it is the email plugin which is missing:
> > ```
> > Error: Cannot find module '@strapi/plugin-email/package.json'      
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > All I did before was `npx create-strapi-app@beta strapi-5-beta --ts`
> > Required System information:
> > 
> > * Node.js version: v20.12.2
> > * NPM version: 10.5.0
> > * Strapi version: 5.0.0-beta.5
> > * Database: SQlite
> > * Operating system: macOS 14.4.1
> > * Is your project Javascript or Typescript: Typescript
> 
> This issue still happens in beta6, it is missing this email plugin. When I tried earlier today, there is no beta6 version of the email plugin. I'm staying with beta5 for now.
> 
> You can fix this by installing the plugin manually npm i —save @strapi/plugin-email@5.0.0-beta.5

I've tried the beta.5 but having this Error: `Cannot find module '@strapi/email/package.json'`

**Comment by Sam-Flexiti** [comment]
> > > I guess I'm having the same issue with a fresh install of strapi@beta. But for me it is the email plugin which is missing:
> > > ```
> > > Error: Cannot find module '@strapi/plugin-email/package.json'      
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > All I did before was `npx create-strapi-app@beta strapi-5-beta --ts`
> > > Required System information:
> > > 
> > > * Node.js version: v20.12.2
> > > * NPM version: 10.5.0
> > > * Strapi version: 5.0.0-beta.5
> > > * Database: SQlite
> > > * Operating system: macOS 14.4.1
> > > * Is your project Javascript or Typescript: Typescript
> > 
> > 
> > This issue still happens in beta6, it is missing this email plugin. When I tried earlier today, there is no beta6 version of the email plugin. I'm staying with beta5 for now.
> > You can fix this by installing the plugin manually npm i —save @strapi/plugin-email@5.0.0-beta.5
> 
> I've tried the beta.5 but having this Error: `Cannot find module '@strapi/email/package.json'`

I think I saw this error when I mixed the versions of Strapi's beta. Since there was a delay getting beta6 of the email plug-in I set it to beta5 in the package.json. Then when I got this error I moved them all back to beta 5 to get it working. Now beta6 builds & deploys fine locally. However I find it is very buggy when using our Postgres DB as the persistence layer. I can't configure the content. 

![Screenshot at May 03 12-57-14](https://github.com/strapi/strapi/assets/162635154/63ec85e2-915e-4000-809e-e2145882ced3)
![Screenshot at May 03 12-56-19](https://github.com/strapi/strapi/assets/162635154/54e1fbbc-aa17-4c12-bf21-eb9f75fe7346)
 
 
 I don't understand why it says "no permissions to see this field". I'm a super admin...

**Comment by hanpaine** [comment]
This issues seems to not occur anymore, we believe it was a releases issue that has now been resolved in a later Beta

**Comment by Chudroy** [comment]
also having this problem on 5.7.0
